javascript/builtins/intl/DateTimeFormat.json

```diff
@@ -313,9 +171,9 @@
                 }
               }
             },
-            "iana_time_zone_names": {
+            "options_parameter": {
               "__compat": {
-                "description": "IANA time zone names in <code>timeZone</code> option",
+                "description": "<code>options</code> parameter",
```

Comment:
Note, this just copies the parent compat info - assumption is that this was added in first version